test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -10724,7 +10727,12 @@ TEST_F(AtenXlaTensorTest, TestNanToNum) {
     ForEachDevice([&](const torch::Device& device) {
       torch::Tensor xla_input = CopyToDevice(input, device);
       torch::Tensor xla_output = torch::nan_to_num(xla_input);
-      AllClose(output, xla_output);
+      if (scalar_type != torch::kDouble) {
+        AllClose(output, xla_output);
+      } else {  // inf entries differ since we convert double to float on TPU
+        AllEqual(output[0], xla_output[0]);
```

Comment:
Is there a way I could test the rest of the entries without getting an unequal value error? @JackCaoG

Reply:
Explained above.